setup-tests-pipeline.ps1

```diff
@@ -17,13 +17,18 @@ if(!$skipCliDownload)
 
   Write-Host "Downloading Functions Core Tools...."
   Invoke-RestMethod -Uri 'https://functionsclibuilds.blob.core.windows.net/builds/3/latest/version.txt' -OutFile version.txt
-  Write-Host "Using Functions Core Tools version: pgopa-CLI"
   $version = "$(Get-Content -Raw version.txt)"
   Remove-Item version.txt
 
-  if ($version -and $version.trim())
+  if (-not($version -and $version.trim()))
   {
     $env:CORE_TOOLS_URL = "https://pgopafunctestv2storage.blob.core.windows.net/cli/pgopa-cli.zip"
+    Write-Host "Using Functions Core Tools version: pgopa-CLI"
+  }
+  else
+  {
+    $env:CORE_TOOLS_URL = "https://functionsclibuilds.blob.core.windows.net/builds/3/latest/Azure.Functions.Cli.win-x64.zip"
```

Comment:
You are changing the core tools from official blob? Looks great.

Reply:
Correct, We are getting the latest release version